src/components/help/HelpDirective.js

```diff
@@ -24,15 +24,17 @@
    * <span ga-help="12,13,14"></div>
   */
   module.directive('gaHelp',
-      function($rootScope, gaHelpService, gaPopup) {
+      function($rootScope, $sce, gaHelpService, gaPopup) {
         var popupContent = '<div class="ga-help-content" ' +
                                 'ng-repeat="res in options.results">' +
-                             '<h2 ng-bind="res[1]"></h2>' +
-                             '<div ng-bind="res[2]"></div>' +
+                             '<h2 ng-bind-html=trustAsHtml(res[1])></h2>' +
+                             '<div ng-bind-html=trustAsHtml(res[2])></div>' +
                              '<img ng-src="{{res[4]}}" ' +
                                   'draggable="false"/>' +
                            '</div>';
 
+        $rootScope.trustAsHtml = $sce.trustAsHtml;
```

Comment:
Usually we avoid attaching function to rootScope when it's not necessary, may be you can apply the trustAsHtml function when you receive results [here](https://github.com/geoadmin/mf-geoadmin3/blob/poa_html_interpretation_fix/src/components/help/HelpDirective.js#L106) 

```
results.push([
  $sce.trustAsHtml(res.rows[0][1],
  $sce.trustAsHtml(res.rows[0][2])
]);
                      
```